src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -85,11 +85,13 @@ export function navigateTo(uri: string, forceLoad: boolean, replace: boolean = f
     const temporaryUri = uri + '?';
     history.replaceState(null, '', temporaryUri);
     location.replace(uri);
-  } else if (replace){
-    history.replaceState(null, '', absoluteUri)
```

Comment:
This code path made no sense (because it wouldn't cause an actual navigation) and appears never to have been used.